assets/src/blocks/CarouselHeader/CarouselControls.js

```diff
@@ -1,38 +1,66 @@
+const {useMemo} = wp.element;
 const {__} = wp.i18n;
 
 export const CarouselControls = ({
   goToPrevSlide = () => {},
   goToNextSlide = () => {},
   goToSlide = () => {},
+  handleAutoplay = () => {},
   currentSlide = null,
   slides = null,
-}) => slides.length > 1 && (
+  autoplay,
+  disableControls,
+}) => useMemo(() => (
   <>
     {/* Arrows */}
-    <button className="carousel-control-prev" onClick={goToPrevSlide}>
+    <button className="carousel-control-prev" onClick={goToPrevSlide} aria-label="Go to previous slide">
       <span className="carousel-control-prev-icon" aria-hidden="true"><i></i></span>
       <span className="visually-hidden">{__('Previous', 'planet4-blocks')}</span>
     </button>
-    <button className="carousel-control-next" onClick={goToNextSlide}>
+    <button className="carousel-control-next" onClick={goToNextSlide} aria-label="Go to next slide">
       <span className="carousel-control-next-icon" aria-hidden="true"><i></i></span>
       <span className="visually-hidden">{__('Next', 'planet4-blocks')}</span>
     </button>
     {/* Indicators */}
     <div className="carousel-indicators-wrapper">
       <div className="container">
-        <ol className="carousel-indicators">
+        <ol className="carousel-indicators" tabIndex={-1}>
           {
-            slides.map((slide, index) =>
+            slides.map((_, index) =>
               <li
-                onClick={() => goToSlide(index)}
                 key={index}
-                className={index === currentSlide ? 'active' : ''}
-                role="presentation"
-              ></li>
+                {...(index === currentSlide) ? {
+                  className: 'active',
+                } : null}
+              >
+                <button
+                  onClick={() => {
+                    if(index !== currentSlide) {
+                      goToSlide(index);
+                    }
+                  }}
+                  tabIndex={0}
+                  onKeyDown={e => {
+                    if ((e.key === 'Enter' || e.key === ' ') && index !== currentSlide) {
+                      e.preventDefault();
+                      goToSlide(index);
+                    }
+                  }}
+                  aria-label={__('Go to slide', 'planet4-blocks') + ` ${index + 1}`}
```

Comment:
We should update this translation string to take the slide number as parameter, rather than adding it at the end. Depending on the language that might not be correct, plus it will be more clear for translators that a slide number is included in the sentence.